### PR TITLE
Clean-up of the gem

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer: 

/cc @grosser 

We need to bring this repo in the zendesk org...

### CODE REVIEW

File: gemfiles/rails42.gemfile [hunk 1]
```diff
@@ -1,5 +0,0 @@
-source "https://rubygems.org"
-
-gemspec path: '../'
-
-gem "activerecord", "~> 4.1.0"
```
Comment (pschambacher) [inline gemfiles/rails42.gemfile]: Just noticed this ~_~

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 test/db
 Gemfile.lock
+gemfiles/*.gemfile.lock
```
Comment (grosser) [inline .gitignore]: how about we go for the delete lock on travis method ... not a big fan of removing those lock files, makes coming back to old projects a [pain](http://grosser.it/2015/08/14/check-in-your-gemfile-lock) :)

**Comment by grosser** [comment]
:+1:

**Comment by pschambacher** [comment]
@grosser then merge it, I have no right in this repo :smile:

File: .gitignore (hunk 2 above)
Comment (grosser) [inline .gitignore]: see just got 

```
Bundler could not find compatible versions for gem "minitest":
  In snapshot (Gemfile.lock):
    minitest (= 5.8.0)

```

showstopper if you are a new ruby developer ...

**Pull request merged**